frontend/src/layout/GlobalModals.tsx

```diff
@@ -100,7 +99,6 @@ export function GlobalModals(): JSX.Element {
             <CreateProjectModal isVisible={isCreateProjectModalShown} onClose={hideCreateProjectModal} />
             <CreateEnvironmentModal isVisible={isCreateEnvironmentModalShown} onClose={hideCreateEnvironmentModal} />
             <UpgradeModal />
-            <ConfirmUpgradeModal />
```

Comment:
I don't think you can remove this because it may be used in onboarding so it needs to be mounted here. You should confirm.